### PR TITLE
Github review process

### PR DESCRIPTION
**Pull request by javiercasares**
We added some information about the review process. Two or one approval needed.

### CODE REVIEW

File: get-involved.md
Comment (getsource) [inline get-involved.md:25]: Maybe something that references the PR also being created by someone from the Hosting Team?

Comment (javiercasares) [reply get-involved.md]: I think the new one is more clarifying:

Once a PR is submitted, it will require an approval process. By default, once it is submitted, it will require review and approval by two people from the Hosting team. In cases where there are PRs related to corrections or internal operational elements, created by someone from the Hosting Team, only one review will be required.

**Review by Crixu — APPROVED** [review]

**Pull request merged**
